### PR TITLE
FIX: FIXES ISSUE WITH KEYBOARD NAVIGATION

### PR DESCRIPTION
**Pull request by junaidbinfarooq**
- Fixes an issue related to navigation only working in the left direction when a field is being edited

### CODE REVIEW

File: grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.jsx [hunk 1]
```diff
@@ -5,6 +5,8 @@ const KEY_DELETE = 'Delete';
 const KEY_F2 = 'F2';
 const KEY_ENTER = 'Enter';
 const KEY_TAB = 'Tab';
+const KEY_ARROW_Left = 'ArrowLeft';
```
Comment (StephenCooper) [inline grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.jsx:8]: Would you mind renaming to `KEY_ARROW_LEFT` to be consistent with the casing. Same for right variable too.

**Comment by StephenCooper** [comment]
Thanks for spotting and fixing this! Just a small nit pick and then I will merge.

File: grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.jsx (hunk 1 above)
Comment (StephenCooper) [inline grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.jsx:8]:
```suggestion
const KEY_ARROW_LEFT = 'ArrowLeft';
```

File: grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.jsx [hunk 2]
```diff
@@ -62,7 +64,7 @@ export default memo(forwardRef((props, ref) => {
     const cancelBeforeStart = props.charPress && ('1234567890'.indexOf(props.charPress) < 0);
 
     const isLeftOrRight = event => {
-        return ['ArrowLeft', 'ArrowLeft'].indexOf(event.key) > -1;
+        return [KEY_ARROW_Left, KEY_ARROW_Right].indexOf(event.key) > -1;
```
Comment (StephenCooper) [inline grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.jsx:67]:
```suggestion
        return [KEY_ARROW_LEFT, KEY_ARROW_RIGHT].indexOf(event.key) > -1;
```

File: grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.jsx [hunk 3]
```diff
@@ -5,6 +5,8 @@ const KEY_DELETE = 'Delete';
 const KEY_F2 = 'F2';
 const KEY_ENTER = 'Enter';
 const KEY_TAB = 'Tab';
+const KEY_ARROW_Left = 'ArrowLeft';
+const KEY_ARROW_Right = 'ArrowRight';
```
Comment (StephenCooper) [inline grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.jsx:9]:
```suggestion
const KEY_ARROW_RIGHT = 'ArrowRight';
```

File: grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.tsx [hunk 4]
```diff
@@ -5,6 +5,8 @@ const KEY_DELETE = 'Delete';
 const KEY_F2 = 'F2';
 const KEY_ENTER = 'Enter';
 const KEY_TAB = 'Tab';
+const KEY_ARROW_Left = 'ArrowLeft';
+const KEY_ARROW_Right = 'ArrowRight';
```
Comment (StephenCooper) [inline grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.tsx:9]:
```suggestion
const KEY_ARROW_LEFT = 'ArrowLeft';
const KEY_ARROW_RIGHT = 'ArrowRight';
```

File: grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.tsx [hunk 5]
```diff
@@ -62,7 +64,7 @@ export default memo(forwardRef((props: ICellEditorParams, ref) => {
     const cancelBeforeStart = props.charPress && ('1234567890'.indexOf(props.charPress) < 0);
 
     const isLeftOrRight = (event: any) => {
-        return ['ArrowLeft', 'ArrowLeft'].indexOf(event.key) > -1;
+        return [KEY_ARROW_Left, KEY_ARROW_Right].indexOf(event.key) > -1;
```
Comment (StephenCooper) [inline grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.tsx:67]:
```suggestion
        return [KEY_ARROW_LEFT, KEY_ARROW_RIGHT].indexOf(event.key) > -1;
```

**Review by StephenCooper — APPROVED** [review]

File: grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.jsx (hunk 1 above)
Comment (junaidbinfarooq) [reply grid-packages/ag-grid-docs/documentation/doc-pages/component-cell-editor/examples/component-editor-2/numericEditor_reactFunctional.jsx]: Oh! Thought I wrote it in all caps. Didn't notice.

**Pull request merged**